### PR TITLE
Docsite improvements/editing

### PR DESCRIPTION
**Pull request by AleksandrHovhannisyan**
(Very cool library! 😄)

I hope you don't mind unsolicited PRs like this. Please let me know if you don't like any changes, and I can roll them back. All changes are related to the docsite:

- Minor typo fix
- General copy editing/cleanup
- Removing text justification

For text justification, I mainly removed it because it makes certain text harder to read, e.g.:

image

### CODE REVIEW

File: docsite/index.html [hunk 1]
```diff
@@ -2182,7 +2178,7 @@ <h6>Push Out</h6>
     <section id="sizes">
       <header>
         <h2>Sizes</h2>
-        <p>No <code class="language-css">px</code> here, relative units all the way. 0's are unsafe but common negative values, while 1-N are safe.</p>
+        <p>No <code class="language-css">px</code> here, relative units all the way. <code>0</code>s are unsafe but common negative values, while 1-N are safe.</p>
```
Comment (AleksandrHovhannisyan) [inline docsite/index.html:2181]: I had trouble understanding what was meant by the second sentence. I think it's worth clarifying this.

File: docsite/index.html [hunk 2]
```diff
@@ -109,13 +109,13 @@ <h1><span>CSS</span> <span>variables.</span></h1>
             <svg viewBox="0 0 24 24">
               <use href="#checkmark-icon"/>
             </svg>
-            Create consistent components
+            Consistent styles and conventions
```
Comment (AleksandrHovhannisyan) [inline docsite/index.html:112]: Mainly changed this for parallelism so it's consistent with the other bullets, feel free to propose an alternative!

File: docsite/index.html [hunk 3]
```diff
@@ -264,28 +264,28 @@ <h2>Why use Open Props?</h2>
...
           <dt>
             <svg  viewBox="0 0 24 24" >
               <path d="M11 3.055A9.001 9.001 0 1020.945 13H11V3.055z" />
               <path d="M20.488 9H15V3.512A9.025 9.025 0 0120.488 9z" />
             </svg>
-            Easy Adoption
+            Incrementally Adoptable
           </dt>
-          <dd>Grab a few props, all the props, props as JS, or adopt only what you use.</dd>
+          <dd>Grab all the props, props as JS, or only what you need.</dd>
```
Comment (AleksandrHovhannisyan) [inline docsite/index.html:288]: I also had a question about this point:

> Map the props from JS or customize builds from the command line

I think it's worth clarifying the wording here. When we say `map the props from JS`, does this mean you can access Open Props's custom properties in JavaScript as variables?

File: docsite/index.css [hunk 4]
```diff
@@ -44,10 +44,6 @@ body {
   }
 }
 
-:not(blockquote) > p {
-  text-align: justify;
-}
-
```
Comment (argyleink) [inline docsite/index.css:50]: def a style choice, i'd like to leave it as is.

File: docsite/index.html (hunk 2 above)
Comment (argyleink) [reply docsite/index.html:112]: thanks for offering a suggestion here, but i prefer how it was..!

File: docsite/index.html [hunk 5]
```diff
@@ -109,13 +109,13 @@ <h1><span>CSS</span> <span>variables.</span></h1>
...
             </svg>
-            Create consistent components
+            Consistent styles and conventions
           </li>
           <li>
             <svg viewBox="0 0 24 24">
               <use href="#checkmark-icon"/>
             </svg>
-            Useful in any framework
+            Compatible with any framework
```
Comment (argyleink) [inline docsite/index.html:118]: this is nice

File: docsite/index.html (hunk 3 above)
Comment (argyleink) [reply docsite/index.html:288]: yep! all the props are importable as js

File: docsite/index.html [hunk 6]
```diff
@@ -1661,7 +1657,7 @@ <h4>Fluid Font Sizes</h4>
     <section id="easing">
       <header>
         <h2>Easing</h2>
-        <p>5 strengths for the classic <code class="language-css">ease</code>, <code class="language-css">easeout</code>, <code class="language-css">easein</code>, and <code class="language-css">easeinout</code>. Some extra easings too, a space for mock elasticity and physics.</p>
+        <p>Five strengths for the classic <code class="language-css">ease</code>, <code class="language-css">ease-out</code>, <code class="language-css">ease-in</code>, and <code class="language-css">ease-in-out</code> timing functions. Plus some extra easings for mock elasticity and physics!</p>
```
Comment (argyleink) [inline docsite/index.html:1660]: nice, fixing those easing names is great

File: docsite/index.html (hunk 1 above)
Comment (argyleink) [reply docsite/index.html:2181]: sure, and wrapping the unit there helps you understand it better?

**Review by argyleink — CHANGES_REQUESTED** [review]
thanks for this effort! there's certainly nuggets in here i'd like to merge, but they're amongst other things that I dont think are necessary (like changing Javascript to JavaScript). 

it'd help me a bunch if this PR turned into smaller more focused PRs, you willing to break each thought out into it's own PR?

**Comment by AleksandrHovhannisyan** [comment]
@argyleink Sure thing! I'll put up separate fixes.

File: docsite/index.html (hunk 1 above)
Comment (AleksandrHovhannisyan) [reply docsite/index.html:2181]: Sorry, could you clarify that bit? I guess here's what confused me:

- What do we mean by `0s are unsafe`? Or does the sentence read `0s are unsafe (but uncommon) negative values`? (Which is still unclear imo)
- What negative values is it referring to?

**Pull request closed without merging**